weave-js/src/components/Panel2/PanelPanel.tsx

```diff
@@ -206,20 +205,14 @@ export const PanelPanelConfig: React.FC<PanelPanelProps> = props => {
     onMouseMove: onBodyMouseMove,
   } = useScrollbarVisibility();
 
-  const [inspectingRoot, setInspectingRoot] = useState(false);
   const [isOutlineMenuOpen, setIsOutlineMenuOpen] = useState(false);
-  const selectedIsRoot = useMemo(
-    () => selectedPanel.filter(s => s).length === 0,
+  const selectedIsRootLevel = useMemo(
+    () => selectedPanel.filter(s => s).length < 2,
```

Comment:
This `2` smells a bit to me. Can you share your intuition here? Is it possible to have a more generic solution here - for example, can we specify if a child panel is allowed to be edited as part of the config?

Reply:
the 2 is odd: what happens if we just have a group with nested children?

Reply:
Im pretty sure panelpanel operates on the whole config only, so the< 2 is if the path is for main -['main'], varbar -[
'sidebar], or root - [], all other panels will have longer paths, though i agree that maybe i can come up with a better solution that might make more sense for the long run